src/Features/LanguageServer/Protocol/Handler/CodeActions/CodeActionResolveHandler.cs

```diff
@@ -130,22 +131,24 @@ public async Task<LSP.VSCodeAction> HandleRequestAsync(LSP.VSCodeAction codeActi
                     var changedAdditionalDocuments = projectChanges.SelectMany(pc => pc.GetChangedAdditionalDocuments());
 
                     // Changed documents
-                    await AddTextDocumentEditsAsync(
-                        textDocumentEdits, changedDocuments,
-                        applyChangesOperation.ChangedSolution.GetDocument, solution.GetDocument, textDiffService,
-                        cancellationToken).ConfigureAwait(false);
+                    var documentEdits = await ProtocolConversions.ChangedDocumentsToTextDocumentEditsAsync(
```

Comment:
Thank you for this, I was looking at making `AddTextDocumentEditsAsync` reusable on Friday :)